server/src/main.go

```diff
@@ -102,6 +102,13 @@ func main() {
 				Required: false,
 				Value:    "/",
 			}),
+			altsrc.NewBoolFlag(&cli.BoolFlag{
+				Name:     "enable-anonymous-login",
+				EnvVars:  []string{"SCRUMLR_ENABLE_ANONYMOUS_LOGIN"},
+				Usage:    "enables/disables the login of anonymous clients",
+				Required: false,
+				Value:    true,
+			}),
```

Comment:
since anonymous login being enable should probably be the default, wouldn't it be more logical to have a flag to disable it? -> 'disable-anonymous-login'

Reply:
it's a matter of semantics but yeah I agree